lib/rules/filename-is-valid-accessible-name.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
we need a selector for this rule or it will have terrible performance for no reason